torch/csrc/jit/codegen/cuda/manager.cpp

```diff
@@ -209,14 +210,32 @@ void runCudaFusionGroup(const Node* fusion_node, Stack& stack) {
   FUSER_PERF_SCOPE("nvFuser::Manager::runCudaFusionGroup");
 
   // Fallback to use if anything goes wrong
-  auto take_fallback = [&]() {
+  auto take_fallback = [&](Stack& stack) {
     // copying graph here since we are eliminating shape information;
     auto copied_graph = fusion_node->g(attr::Subgraph)->copy();
     EraseShapeInformation(copied_graph);
     enableAliasCopyNodes(copied_graph, copied_graph->block());
     InterpreterState{Code(copied_graph, "fallback_cuda_fuser")}.run(stack);
```

Comment:
We should really cache and save the `InterpreterState` code here. This fallback actually does get run in real networks, and this whole processes happens on each invocation

Reply:
That probably shouldn't be in this pr - but something to do soon